README.md

```diff
@@ -72,12 +72,34 @@ To customize or regenerate the benchmarks as well as to see other benchmark resu
 
 # Development
 
-## Running tests
+## Running Python Tests
 
 ```
 make test
 ```
 
+## Debugging and Running C++ Code
+
+For debugging and running C++ code independently from python module, CMake config files are provided in pyvptree/CMakeLists.txt.
+For generating C++ makefiles and building tests run:
+
+```
+mkdir build
+cmake -G "Unix Makefiles" ../pyvptree
+make
```

Comment:
there's a cd build missing here is it?

Reply:
I have changed it to build all with Makefile now:
```
make cpp-test
```